### PR TITLE
Add README.MD

### PR DESCRIPTION
**Pull request by stephanie-fu**
Added a README with a few screenshots and a list of possible unit conversions.

### CODE REVIEW

**Comment by benz-le** [comment]
How did you get the image url for your screenshot?

**Comment by stephanie-fu** [comment]
I took the screenshots on Haiku myself and added them to images/

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Vandelay
```
Comment (humdingerb) [inline README.MD]: From the About, it looks like it's actually "VanDelay". Mmore occurances below...

File: README.MD [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+![About Vandelay](images/About.png)
```
Comment (humdingerb) [inline README.MD]: Not sure the About window is that informative. How about just the main window shot between the next paragraph and the "Available units:"?
Also, I'd prefer to have the screenshot including the window border. The yellow tabs are a hallmark of Haiku and should be signaled whenever possible to create Haiku-awareness. :)

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
You think you could move the screenshot out of "images" subfolder and adjust the link in the readme?
Having an extra folder for just that one screenshot is a bit overkill.

**Comment by stephanie-fu** [comment]
True lol - I wasn't sure if it was an organization thing that every app should have because I modeled it off of another app that had more screenshots.

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**